src/chi/components/divider/divider.scss

```diff
@@ -0,0 +1,46 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .a-divider {
+    border-top: 0.0625rem solid set-color(grey, 20);
+    margin: 0.5rem 0;
+
+    &.-dark {
+       border-top: 0.0625rem solid set-color(grey, 30);
+    }
+
+    &.-inverse {
+      border-top: 0.0625rem solid rgba(set-color(white), 0.2);
+    }
+
+    &.-vertical {
+      border-right: 0.0625rem solid set-color(grey, 20);
+      border-top: 0;
+      height: 1.75rem;
```

Comment:
@mattnickles height must be fixed?

Reply:
I think so

Reply:
height: 100%. However, it will require the parent container to have a height specified.

Reply:
Height:100% don't works if i use a height specified in parent container, but height:inherit works fine